web/src/js/ducks/settings.js

```diff
@@ -0,0 +1,92 @@
+import { StoreCmds } from '../actions'
+import { addLogEntry } from './eventLog'
+
+export const WS_MSG_TYPE = 'settings'
+export const WS_MSG_CMD_RESET = 'reset'
+export const WS_MSG_CMD_UPDATE = 'update'
```

Comment:
They are two different things. `RESET` is an action type and `WS_MSG_CMD_RESET` is a config option. I think keeping them separate can improve maintainability.